language-avro-0.1.2.0

`language-avro-0.1.2.0` fails to compile on the current nightly snapshot with the following error:
```
--  While building package language-avro-0.1.2.0 using:
      /var/stackage/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_3.0.1.0_ghc-8.8.3 --builddir=.stack-work/dist/x86_64-linux/Cabal-3.0.1.0 build lib:language-avro test:avro-parser-haskell-test --ghc-options ""
    Process exited with code: ExitFailure 1
    Logs have been written to: /var/stackage/work/unpack-dir/.stack-work/logs/language-avro-0.1.2.0.log

    Preprocessing library for language-avro-0.1.2.0..
    Building library for language-avro-0.1.2.0..
    [1 of 2] Compiling Language.Avro.Types
    
    /var/stackage/work/unpack-dir/unpacked/language-avro-0.1.2.0-b6cffdfeff894f4ffa8ada3c600d7f39bc6953f5aa00e8a048df0ab45dcd7763/src/Language/Avro/Types.hs:9:1: error:
        Could not find module ‘Data.Avro.Schema’
        Perhaps you meant
          Data.Aeson.Schema (needs flag -package-key aeson-schemas-1.0.3)
          Data.YAML.Schema (needs flag -package-key HsYAML-0.2.1.0)
        Use -v (or `:set -v` in ghci) to see a list of the files searched for.
      |
    9 | import Data.Avro.Schema
      | ^^^^^^^^^^^^^^^^^^^^^^^
    
    /var/stackage/work/unpack-dir/unpacked/language-avro-0.1.2.0-b6cffdfeff894f4ffa8ada3c600d7f39bc6953f5aa00e8a048df0ab45dcd7763/src/Language/Avro/Types.hs:10:1: error:
        Could not find module ‘Data.Avro.Types.Value’
        Use -v (or `:set -v` in ghci) to see a list of the files searched for.
       |
    10 | import Data.Avro.Types.Value
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
cc @kutyel

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -1344,7 +1344,7 @@ packages:
         - language-protobuf
 
     "Flavio Corpa <flaviocorpa@gmail.com> @kutyel":
-        - language-avro < 0 # avro-0.5.0.0 https://github.com/commercialhaskell/stackage/issues/5262
+        - language-avro
 
     "Matvey Aksenov <matvey.aksenov@gmail.com> @supki":
         - terminal-size

